patroni/__init__.py

```diff
@@ -50,19 +55,20 @@ def initialize(self):
 
         # is data directory empty?
         if self.postgresql.data_directory_empty():
-            # racing to initialize
-            if self.ha.dcs.race('/initialize'):
-                self.postgresql.initialize()
-                self.ha.dcs.take_leader()
-                self.postgresql.start()
-                self.postgresql.create_replication_user()
-                self.postgresql.create_connection_users()
-            else:
-                while True:
```

Comment:
Hmm,
why we are not waiting for leader anymore?

And what if initialization failed?